docs/content/components/Testimonial.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: Testimonial
```

Comment:
Should Stack also have a documentation page?

Reply:
Yeah, let me add one in. I was originally keeping it as an internal helper component, with examples in Storybook but makes sense to have it in main docs too 👍